server/src/utils/enums/Validations.ts

```diff
@@ -1,4 +1,6 @@
 export enum Validations {
 	PASSWORD_REGEX = '^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{8,})',
+	EMAIL_REGEX = '^[^@]+@[^@]+$',
```

Comment:
We need to use this regex because on the front, I check email with this eregex, and it will be fishy to check email with other rules on the back... Unfortunately, we don't know exactly what does @isEmail decorator.

I will use @Matches, this is more safe